src/controllers/post.js

```diff
@@ -7,7 +10,33 @@ export const create = async (req, res) => {
     return sendDataResponse(res, 400, { content: 'Must provide content' })
   }
 
-  return sendDataResponse(res, 201, { post: { id: 1, content } })
+  const auth = req.headers.authorization
```

Comment:
don't repeat the token validation code here, there's a middleware function called `validateAuthentication` that should be called in the router, before this controller function. In fact, the validation is already being called in the post router:

```js
router.post('/', validateAuthentication, create)
```